### ISSUE TITLE
Screenshot

### ISSUE THREAD

**Issue by MartyIX** [0xc3u/Indiko.Maui.Controls.Markdown#26]
Hi,

would it be a good idea to add a screenshot of the control in the readme?

Cheers!

**Comment by 0xc3u** [comment]
Yes, why not? Please create a pull request with the updated README, including two screenshots from both platforms.

**Comment by 0xc3u** [comment]
![Simulator Screenshot - iPhone 16 - 2024-12-06 at 11 27 21](https://github.com/user-attachments/assets/70aba445-79f7-4adf-90b4-2b6e1e9dd227)
